utils/rpms/daos.spec

```
@@ -562,8 +562,11 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Wed Apr  6 2022 Johann Lombardi <johann.lombardi@intel.com> 2.3.100-1
+- Switch version to 2.3.100 for 2.4 test builds
+
 * Wed Apr  6 2022 Joseph Moore <joseph.moore@intel.com> 2.1.100-22
```

Oops.  This landed before we got to fixing it:
```suggestion
* Wed Apr  6 2022 Joseph Moore <joseph.moore@intel.com> 2.1.100-26
```
We should fix in this PR.  This will need fixing on release/2.2 also if that was branched with this in it with whatever PR to use to initialise release/2.2